src/platform/backends/lxd/lxd_vm_image_vault.cpp

```diff
@@ -56,11 +72,71 @@ auto parse_percent_as_int(const QString& progress_string)
 
     return -1;
 }
+
+QString post_process_downloaded_image(const QString& image_path, const mp::ProgressMonitor& monitor)
+{
+    QString new_image_path{image_path};
+
+    if (image_path.endsWith(".xz"))
+    {
+        new_image_path = mp::vault::extract_image(image_path, monitor, true);
+    }
+
+    QString original_image_path{new_image_path};
+    new_image_path = mp::backend::convert_to_qcow_if_necessary(new_image_path);
```

Comment:
You know, I never tried just passing the image as-is.  I can certainly try it with `lxc` and see what happens.  If it works, great!

Reply:
After some experimenting, `lxc` will import the `xz` image, but treats it as if it were a compressed `rootfs` and not a VM image.  I see no way to force it otherwise.  Also, if we leave it raw, it is a **huge** upload that takes more time than having `qemu-img` convert it.  I think we are stuck with this.

Reply:
I believe we can safely resolve this since Mr. LXD said it only supports qcow2 :smiley: